rholang/src/main/tree_sitter/grammar.js

```diff
@@ -125,18 +117,20 @@ module.exports = grammar({
             $.percent_percent,
             $.mult,
             $.div,
+            $.mod,
             $.not,
             $.neg,
             $.method,
-            $._parenthesized,
+            $.parenthesized,
```

Comment:
I think because if I left it underscored, I would have to add a match case where if a node is `(`, then parse it's sibling. It just seemed error prone to match against a single `(`. The flow of my code is that I know exact positions of node kinds and if I come across an unexpected node, I throw an error and not continue over.

Reply:
My point is - you should almost never match on single lexems, such as '(' - you should almost invariably use `named_child` and friends.